includes/Handlers/Products.php

```diff
@@ -577,6 +583,117 @@ public function set_synced_with_square( $product ) {
 		// phpcs:enable WordPress.Security.NonceVerification.Missing
 	}
 
+	/**
+	 * Sets up stock change tracking for bulk/quick edit operations.
+	 *
+	 * This hooks into the stock change events to capture previous values and sync to Square.
+	 *
+	 * @since 4.9.0
```

Comment:
```suggestion
	 * @since x.x.x
```

We are using x.x.x version for the PR and it will get replace to actual version during the release process. Same applies to other places as well.